code/_onclick/click.dm

```diff
@@ -98,7 +98,7 @@
 		CtrlClickOn(A)
 		return
 
-	if(incapacitated(ignore_restraints = 1))
+	if(incapacitated(ignore_restraints = TRUE, ignore_stasis = TRUE))

```

Comment:
This is in `ClickOn`. I'm not sure what other ramifications this could have, but I can't imagine much.